x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/searchable_snapshot_field/searchable_snapshot_field.tsx

```diff
@@ -24,8 +24,16 @@ export interface Props {
   phase: 'hot' | 'cold' | 'frozen';
   canBeDisabled?: boolean;
 }
+export interface Links {
+  fullyMountedSearchableSnapshotLink: string;
+  partiallyMountedSearchableSnapshotLink: string;
+}
 
-const geti18nTexts = (phase: Props['phase']) => {
+const geti18nTexts = (
+  phase: Props['phase'],
+  link1: Links['fullyMountedSearchableSnapshotLink'],
```

Comment:
nit: It seems the `Links` interface is only used here to extract each link type, is there any reason for that? It seems to me we could simply use the more descriptive variable names from the interface in the `geti18nTexts` arguments definitions

Reply:
Thanks for the advice! I've added deebdb5def3d92a97cf9f8252e8e541c04540cd8, which is hopefully what you had in mind.